extensions/DrawNodeEx/src/DrawNodeEx/DrawNodeEx.cpp

```diff
@@ -48,13 +48,13 @@ NS_AX_EXT_BEGIN
 
 static inline Tex2F v2ToTex2F(const Vec2& v)
 {
```

Comment:
Since tex2f, vec2 is same in axmol, all v2ToTex2F can be removed

Reply:
done